outputs/core_pools.json

```diff
@@ -81,7 +80,6 @@
   },
   "base": {
     "0xc771c1a5905420daec317b154eb13e4198ba97d0000000000000000000000023": "rETH-WETH-BPT",
-    "0xfb4c2e6e6e27b5b4a07a36360c89ede29bb3c9b6000000000000000000000026": "cbETH/WETH",
```

Comment:
Why is this vanishing?

Reply:
Also fell under, but base is interesting: https://app.balancer.fi/#/base/pool/0xfb4c2e6e6e27b5b4a07a36360c89ede29bb3c9b6000000000000000000000026

Basically as things make enough fees they get all the GOLD bags, and so then their deposits rise/fall with whoever got gold bags vs who didn't in the whole core pools math.
This chain is another interesting edge case.

Reply:
tvl is only ~$156k